tools/depends/.gitignore

```diff
@@ -14,6 +14,7 @@
 /target/*/native/*
 /target/*/x86/*
 /target/*/x86_64-linux-gnu-*/*
+/target/*/x86*-linux-android-*/*
```

Comment:
Why not x86_64-linux-android?

Reply:
The x86 itself was also not prevented, with them both stays away

Reply:
Which x86?